start_refresh.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+curl \
+--header "Accept: application/json" \
+--header "Content-Type: application/json" \
+--request POST $REFRESH_SCRIPT_URL
```

Comment:
Headers are most likely unnecessary as we're not passing anything and we do not parse output.
`curl -X POST $REFRESH_SCRIPT_URL` should be enough